app/views/businessmatching/updateservice/add/fit_and_proper.scala.html

```diff
@@ -41,8 +41,8 @@
             heading = "businessmatching.updateservice.fitandproper.heading",
             section = "summary.updateservice",
             panel = false,
-            supportingContent = supportingContent
-
+            supportingContent = supportingContent,
+            classes = Seq("inline")
```

Comment:
Additional ratio button (outside of ticket) where the buttons were stacked vertically despite no conditional reveal, this is now corrected.